src/NUnitTestExtractor/NUnitTestExtractor/NUnitTestExtractorApp.cs

```diff
@@ -109,7 +103,7 @@ private static void WriteTestDllAndName(StreamWriter writer, string dll, string
         /// <param name="level">Specifies the level of granuality to use for the output: either namepspace, class or function</param>
         /// <param name="writer">The StreamWriter which will be passed to WriteTestDllAndName() </param>
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Reliability", "CA2001:AvoidCallingProblematicMethods", MessageId = "System.Reflection.Assembly.LoadFrom")]
-        private static void GetTests(IList<string> dlls, Level level, StreamWriter writer)
+        public static void GetTests(IList<string> dlls, Level level, StreamWriter writer)
```

Comment:
Code Analysis says you need to check dlls for null before using it.
You can either import the Utilities project that NTestController uses and then add this line in this function:
`ThrowIf.ArgumentNull(dlls, nameof(dlls));`
or just check for null and throw an exception if it's null.